packages/js/client/src/plugin/PluginWrapper.ts

```diff
@@ -78,9 +78,7 @@ export class PluginWrapper extends Wrapper {
       await this._sanitizeAndLoadEnv(client, module);
 
       let jsArgs: Record<string, unknown>;
-
-      // If the args are a msgpack buffer, deserialize it
-      if (args instanceof ArrayBuffer) {
+      if (args instanceof ArrayBuffer || args instanceof Uint8Array) {
```

Comment:
Uint8Array is just one type of `ArrayBufferView`. I propose we change `args instanceof Uint8Array` to `ArrayBuffer.isView(args)`.